sky/engine/core/window/window.cc

```diff
@@ -14,6 +14,12 @@
 namespace blink {
 namespace {
 
+// These strings must match the corresponding strings in hooks.dart
+const std::map<sky::AppLifecycleState, const char *> kLifecycleStateMap = {
```

Comment:
I figured it would be easier to keep the strings in sync between the C++ and Dart sides (and to debug if something gets out of sync)

I can change this to an int if that's preferable.